feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/README.md

```diff
@@ -0,0 +1,40 @@
+# RT-2.10: IS-IS change LSP lifetime
+
+## Summary
+
+* Changing the lsp lifetime and verifying isis lsp parameters
+
+## Procedure
+
+    * Configure IS-IS for ATE port-1 and DUT port-1.
+    * Modify the default lifetime of the LSP PDU.
+    * The default lifetime of the LSP PDU is 1200 seconds.
+        This parameter can be updated using the LSP lifetime parameter.
+        LSP lifetime indicates how long the LSP PDU originated by the DUT should remain in the network. 
+        The DUT regenerates the LSP PDU typically ~300 seconds before its expiration.
```

Comment:
I think here assumption is LSP refresh interval is dynamically calculated from the LSP lifetime ?
Based on implementation , LSP refresh time can also be after a specific interval.

Reply:
@arvbaska1, can you please address these requests in a different pull? We need to merge this one soon.